state-chain/pallets/cf-environment/src/lib.rs

```diff
@@ -568,6 +564,30 @@ impl<T: Config> Pallet<T> {
 				}),
 		}
 	}
+
+	fn calculate_utxos_and_change(spendable_utxos: Vec<Utxo>) -> Option<(Vec<Utxo>, BtcAmount)> {
```

Comment:
This method doesn't need to return the spendable_utxos (it's always the same as the input value).

Also the name is a bit misleading - it calculates the change specifically for the consolidation transaction.

We could also just change the `select_utxos_for_consolidation` such this it returns the change amount too.

Reply:
Refactored